lib/signable/template.rb

```diff
@@ -17,5 +17,9 @@ def update
     def delete
       raise "not available"
     end
+
+    def remind
+      raise "not available"
+    end
```

Comment:
I've overridden the `Client `remind` in this subclass because it doesn't apply for Signable templates in the way Smart use them.

Reply:
this gem is not only for `smart`

Reply:
I meant "Smart" as in our organisation, not the `smart` platform.
But all platforms share the same codebase, and our codebase only makes two calls to this `Template`, one `find` and one `all`.

Reply:
why implement the method?

Reply:
To follow the existing pattern in this class of implementing `save`, `update` and `delete` to override any request wrapper methods available in the superclass which Signable does not support for its Template resource.

When Anthony built this gem, I expect he did this because he thought raising a simple "not available" error here, in this `Template` class, was better than letting the method call go up to the superclass which includes `Client` and causing it to send a pointless request to the Signable API.  As I understand it, these methods are like guard clauses in case any app that installs this gem tries to use it to send the Signable API requests about its Template resource that it doesn't support.